src/codegen/sys/cargo_toml.rs

```diff
@@ -55,10 +54,21 @@ fn fill_empty(root: &mut Table, env: &Env, crate_name: &str) {
 
     let deps = upsert_table(root, "dependencies");
     for ext_lib in &env.config.external_libraries {
-        let ext_package = format!("{}-sys", ext_lib.crate_name);
+        let ext_package = if ext_lib.crate_name == "cairo" {
+            format!("{}-sys-rs", ext_lib.crate_name)
+        } else if ext_lib.crate_name == "gdk_pixbuf" {
+            format!("gdk-pixbuf-sys")
+        } else {
+            format!("{}-sys", ext_lib.crate_name)
+        };
```

Comment:
@GuillaumeGomez You still want this part and next?

Reply:
Yes. You see an issue with them?

Reply:
This function called only for new crates,
it too linked with gtk-rs,
and IMHO will be changed manually in most cases.
So it not bad but just unneeded.

Reply:
Those crates are specific to gtk-rs anyway. I prefer having them generated nicely instead of the current output.

Reply:
Then I merge

Reply:
Thanks! :)